src/components/notifications/PageNotifications.vue

```diff
@@ -16,10 +16,15 @@
       <div slot="subtitle">Don't worry, they are on their way!</div>
     </TmDataMsg>
 
-    <div
+    <router-link
       v-for="notification in notifications"
       :key="notification.id"
       class="notification"
+      title="Transactions"
+      :to="{
+        name: 'transactions',
+        params: { networkId: notification.link.split('/')[1] },
```

Comment:
```suggestion
        params: { networkId: notification.link.trim('/') },
```

Reply:
It doesn't work 

<img width="300px" src="https://user-images.githubusercontent.com/40721795/82584249-82ca9100-9b94-11ea-9d21-055fa678b8e1.png">

Reply:
upsi a I see know

Reply:
why not `:to="notification.link"`?

Reply:
the logic you are using would only work for transaction events, am I wrong?

Reply:
> why not :to="notification.link"?

It is the first thing I did but then we get the uncaught promise error on navigation. We need to set name and params

Reply:
I don't get that error. The link actually is correct but is blocked somehow as the network doesn't change prior :man-shruggin: